ua/lviv/iot/algo/part1/lab1/Chair.java

```diff
@@ -0,0 +1,57 @@
+package ua.lviv.iot.algo.part1.lab1;
+import lombok.*;
```

Comment:
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad